tasks/main.yml

```diff
@@ -66,7 +66,7 @@
     owner: 'root'
     group: 'root'
     mode: '0755'
-  when: ((ferm__enabled|d() | bool) and (docker__ferment|d() | bool == False))
+  when: docker__ferm_post_hook|d() | bool
```

Comment:
This variable is defined by default in the role, so you don't need to check if it's defined. This can be rewritten as:
```yaml
when: docker__ferm_post_hook|bool
```